NLibrary/Classes/Service Files/FirebaseService.swift

```diff
@@ -262,14 +262,14 @@ public enum APIFailureResult: Error {
     }
 
     public func getUserPlaylistDetails() {
-        guard let currentUser = Auth.auth().currentUser else {
-            return
-        }
-        self.ref.child("Playlists/\(currentUser.uid)")
-            .observe(.value, with: { [weak self] (snapshot) in
-                let value = [snapshot.key: snapshot.value!] as NSDictionary
-            self?.repoPlaylist!.successfulGetUserPlaylists(dictionary: value)
-        })
+//        guard let currentUser = Auth.auth().currentUser else {
+//            return
+//        }
+//        self.ref.child("Playlists/\(currentUser.uid)")
+//            .observe(.value, with: { [weak self] (snapshot) in
+//                let value = [snapshot.key: snapshot.value!] as NSDictionary
+//            self?.repoPlaylist!.successfulGetUserPlaylists(dictionary: value)
+//        })
```

Comment:
You probably commented this out to save it for a later date, maybe consider putting this code in a seperate file locally and then remove the comments. Makes it much easier to read the code